src/client/Microsoft.Identity.Client/PlatformsCommon/Interfaces/IDeviceAuthManager.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client.PlatformsCommon.Interfaces
+{
+    internal interface IDeviceAuthManager
```

Comment:
As discussed above, these objects need to do more so that other objects are less responsible.